lib-multisrc/heancms/src/eu/kanade/tachiyomi/multisrc/heancms/HeanCms.kt

```diff
@@ -343,6 +384,27 @@ abstract class HeanCms(
             summaryOff = intl["pref_show_paid_chapter_summary_off"]
             setDefaultValue(SHOW_PAID_CHAPTERS_DEFAULT)
         }.also(screen::addPreference)
+
+        if (enableLogin) {
+            SwitchPreferenceCompat(screen.context).apply {
```

Comment:
imo instead of another preference, check if both email and password are not empty to use login credentials